src/OnboardingSPA/components/Drawer/DrawerPanel/DesignColors.js

```diff
@@ -375,8 +382,8 @@ const DesignColors = () => {
 				: selectedColorsLocal?.secondary ?? defaultColor;
 		const tertiaryColorTemp =
 			customColors && customColors?.tertiary !== ''
```

Comment:
```suggestion
			customColors && customColors?.['header-background']]!== ''
```
Should we check for `header-background` instead of `tertiary`?